src/neptune/types/series/float_series.py

```diff
@@ -96,3 +103,18 @@ def unit(self):
 
     def __str__(self):
         return "FloatSeries({})".format(str(self.values))
+
+    def clear_of_unsupported_values(self):
+        cleared_values = [
```

Comment:
How about `steps` and `timestamps` here? As we're getting rid of particular value we have to delete corresponding timestamp and step as well

Reply:
True, missed that! both of those lists are now cleared 😄